api/src/util/text_extractor.py

```diff
@@ -15,6 +15,9 @@
 
 logger = logging.getLogger(__name__)
 
+TEXT_EXTRACTOR_LOG_PREFIX = "TextExtractor:"
```

Comment:
Our logs have the file_path/func/line number so we know where they were emitted from - do we need this?